holmes/utils/config_utils.py

```diff
@@ -0,0 +1,69 @@
+"""
+Configuration utility functions for HolmesGPT.
+"""
+
+from typing import Any, Dict, List, Optional
+
+
+def merge_transformer_configs(
```

Comment:
I think that this implementation might be and issue, if i understand correclty it means that we can have only 1 trasformer from the same type. 
In that case i think that we need to break it down and add transformer name + type to the config and that way support more dynamic usage. WDYT?